test/linter/test-consistency.js

```diff
@@ -275,11 +293,13 @@ class ConsistencyChecker {
   }
 
   /**
-   * @param {SupportStatement} compatData
-   * @return {string | null}
+   * Get the version added for a feature or recursively search the array for the earliest version
+   *
+   * @param {SupportStatement} compatData The compat data to process
+   * @returns {?string} The earliest version added in the data
```

Comment:
```suggestion
   * @returns {string | null} The earliest version added in the data
```

Reply:
The `?` ["indicates that the type is either the specified type, or null"](https://jsdoc.app/tags-type.html), so this is actually correct syntax.